conode/README.md

```diff
@@ -1,56 +1,612 @@
+certificates for domains which you control. `certbot` writes the files it
+creates into `/etc/letsencrypt`.
+
+If the user you use to run the conode has the rights to read from the directory
+where Let's Encrypt writes the private key and the current certificate, you can
+arrange for conode to share the TLS certificate used by the server as a whole:
+
+```
+WebSocketTLSCertificate = "/etc/letsencrypt/live/conode.example.com/fullchain.pem"
+WebSocketTLSCertificateKey = "/etc/letsencrypt/live/conode.example.com/privkey.pem"
+```
+
+Let's Encrypt certificates expire every 90 days, so you will need
+to restart your conode when the `fullchain.pem` file is refreshed.
+
+## Run your conode
+
+Once the setup is done with one of the two options, you can finally run your
+conode depending on the previously chosen option.
+
+### Option 1: :computer: Run with the command line
+
+Simply launch the following:
+
+```bash
+$ conode server
+```
+
+It is recommended to use more verbose logging and this can be done with the `-d`
+option. It is also possible to specify a different location than the default one
+for the private.toml file with the `-c` option. If you used Docker to set up your
+conode, then the private.toml file is not in the default location. You can then
+use:
+
+```bash
+$ conode -d 2 -c ~/conode_data/private.toml server
+```
+
+You can use the `-h` for a description of the available commands.
+
+If you want to run the server in the background, you can use the `screen`
+program:
+
+```bash
+screen -S conode -d -m conode server
+```
+
+To enter the screen, type `screen -r conode`. You can detach from it with
+`<ctrl-a> d`.
+
+
+### Option 2: :whale: Run with docker
+
+Type the following to start the conode program with docker:
+
+```bash
+$ docker run --restart always -d -p 7770-7771:7770-7771 --name conode -v ~/conode_data:/conode_data dedis/conode:latest
+```
+
+Because it will run detached, you can use `docker logs -f conode` to see the
+logs. It will be restarted on the next boot as well. To stop it, you can use
+`docker stop`. Then, you must remove it with `docker rm <id>` (the id is found
+with `docker ps -a`).
+
+Pro tip: you can use the following options for docker in order to gracefully
+handle the logs and prevent a disk saturation: 
+`--log-opt max-size=10m --log-opt max-file=4 --log-opt compress=true`
 
-## Creating Your own Cothority
+If you have systemd, you can simply copy the `conode.service` file and add it to
+your systemd-startup. Of course, you should do this as a non-root user:
 
-For most of the apps you need at least 3 running nodes. Once you have them up
+```
+$ wget https://raw.githubusercontent.com/dedis/cothority/conode/conode.service
+$ systemctl --user enable conode.service
+$ systemctl --user start conode
+```
+
+Unfortunately, systemd doesn't allow a user to run a service at system startup,
+and all user services get stopped once the user logs out!
```

Comment:
I think it is possible to run user systemd services at startup, see https://wiki.archlinux.org/index.php/Systemd/User#Automatic_start-up_of_systemd_user_instances

Reply:
I think this section comes from @ineiti, I left it "as is"